ngraph/core/include/ngraph/variant.hpp

```diff
@@ -7,90 +7,30 @@
 #include <map>
 #include <string>
```

Comment:
do we need these headers?

Reply:
yes we need. We use both structures in this header